civihr_default_theme/templates/page/page.tpl.php

```diff
@@ -149,12 +150,8 @@
     <div class="container">
       <div class="text-center">
         Powered by CiviHR <?php print get_civihr_version(); ?>.
-        CiviHR is openly available under the <a href="http://www.gnu.org/licenses/agpl-3.0.html ">GNU AGPL License</a>.
-        <br />
-        <a target="_blank" href="http://civihr-documentation.readthedocs.io/en/latest/">User Guide</a>&nbsp;
-        <a target="_blank" href="https://github.com/civicrm/civihr">Download CiviHR</a>&nbsp;
-        <a target="_blank" href="https://civihr.atlassian.net/wiki/display/CIV/Welcome ">View Wiki page</a>&nbsp;
-        <a target="_blank" href="https://civihr.org">Project website</a>
+        CiviHR is openly available under the <a href="http://www.gnu.org/licenses/agpl-3.0.html">GNU AGPL License</a> and can be downloaded from
```

Comment:
Maybe you should check with Mike, but "downloaded from Project Website" seems a bit strange without a "the"